navigation/DrawerNavigation/CustomDrawerContent.tsx

```diff
@@ -38,11 +38,11 @@ function CustomDrawerContent({ navigation }: Props) {
         <SocialImagesGrid />
 
         {/* Login / Logout and Feedback button beneath the social images */}
-        <View style={{ margin: 10, marginTop: 30, marginLeft: 20 }}>
+        <View style={{ marginTop: 30 }}>
           {!isLoggedIn && <DrawerButton text='LOG IN' icon={<MaterialIcons name="login" size={24} color="black" />} onPress={() => navigation.navigate('LoginScreen')} />}
           {isLoggedIn && <DrawerButton text='LOGOUT' icon={<MaterialIcons name="logout" size={24} color="black" />} onPress={onLogout} />}
         </View>
-        <View style={{ margin: 10, marginLeft: 20 }}>
+        <View style={Platform.OS === 'ios' && { marginTop: 20}}>

```

Comment:
I would change this
`<View style={Platform.OS === 'ios' && { marginTop: 20}}>`

to be platform independent
`<View style={{ marginTop: 20 }}`

You can see in the screenshot below that those two buttons are quite close for me. In your screenshots that is not visible, I think because the Drawer only updates when you restart the app with 'r' in the command line.

<img src="https://github.com/ClimateMind/frontend-native-app/assets/78958483/72c061c7-0902-475a-8598-c26f0fc23baa" width=200/>